ntc_templates/templates/alcatel_sros_show_router_ospf_interface.textfsm

```diff
@@ -0,0 +1,27 @@
+Value Required INTERFACE (\S+)
+Value Required AREA_ID (\S+)
```

Comment:
Looking to match the only other existing OSPF Interface command module: https://github.com/networktocode/ntc-templates/blob/master/ntc_templates/templates/cisco_ios_show_ip_ospf_interface_brief.textfsm.

Either update this one or also update the other module to match `AREA_ID`

```suggestion
Value Required AREA (\S+)
```